File: SoccerPlayerApi/SoccerPlayerApi/Dtos/Facts/ImportFactDto.cs
namespace SoccerPlayerApi.Dtos.Facts
{
    public class ImportFactDto
    {
        public int RowNumber { get; set; }
        public decimal Amount { get; set; }
        public string DataType { get; set; }
        public string Dimension1Aggregation { get; set; }
        public string? Dimension2Aggregation { get; set; }
        public string? Dimension3Aggregation { get; set; }
        public string? Dimension4Aggregation { get; set; }
        public string TimeAggregation { get; set; }
    }
}

File: SoccerPlayerApi/clientapp/src/app/models/imports/ImportFactDto.ts
export interface ImportFactDto {
    RowNumber : number,
    Amount: number;
    DataType: string;
    Dimension1Aggregation: string;
    Dimension2Aggregation: string | null;
    Dimension3Aggregation: string | null;
    Dimension4Aggregation: string | null;
    TimeAggregation: string;
} 
